### DISCUSSION TITLE
Question on: COMMERCIAL SOFTWARE LICENSE AGREEMENT

### DISCUSSION THREAD

**Discussion by WaxTheSky** [AutomaApp/automa#700] [Q&A] [answered]
Hello everyone, hoping I can get some help here. I have been reading over the commercial software licensing agreement and noticed that there are many references to payment and Subscription Fees, but no where is it listed where to pay. Can some one let me know how to pay or where the separation sits since there is no "business.automa.site" to setup commercial payments.

Where I am referring to: 
https://github.com/AutomaApp/automa/blob/main/LICENSE.txt
&
https://www.automa.site/license/commercial

**Comment by Kholid060** [comment]
If you're downloading Automa from the chrome web store or firefox add-ons, you don't have to pay anything.

**Comment by hokiman** [comment, marked as answer]
Hi, 

It's Hokiman from Automa here 😄👋

In the near future, we plan to release a version of Automa that is meant to be used by businesses with added features _(example: collaboration for team)_

there will be a distinction between the "community version" and "business version"

where the exact distinction lies will be decided in the near future—we will try to model this based on metabase (https://github.com/metabase/metabase)

as @Kholid060 said, the version of Automa that everyone can access and use will stay free to use for the community
